lib/mandrill_dm/message.rb

```diff
@@ -276,6 +276,15 @@ def inline_attachments?
       mail.attachments.any?(&:inline?)
     end
 
+    def html_content?
+      return true if mail.content_type.nil?
+      mail.content_type =~ %r{text/html} ? true : false
```

Comment:
Hmm, just playing with the regex... it seems my regex-fu is not great.  That regex would also match `text/html5` although thinking more about this, I don't think it *really* matters.  Alternatively, we could just `mail.content_type[0..8] == 'text/html'` but then we'd need a `nil` check.  If someone has a regex that works for `text/html` and `text/html; ...` that would be great, otherwise, I think my previous regex is good enough.